shared/cloud/ess-getting-started-obs.asciidoc

```diff
@@ -0,0 +1,24 @@
+// Include this file in your docs:
+// include::{docs-root}/shared/cloud/ess-getting-started-obs.asciidoc[]
+
+// To include APM Server instructions, add this attribute:
+// :include-apm-instructions:
+
+. {ess-trial}[Get a free trial].
+
+. Log into {ess-console}[Elastic Cloud].
+
+. Click *Create deployment*.
+
+. Select *Elastic Observability* and give your deployment a name.
+
+ifdef::include-apm-instructions[]
+. Click *Create deployment* and copy the password for the `elastic` user.
+
+. Select *APM* from the menu on the left and make note of the APM endpoint and APM Server secret token.
+You'll need these in step two.
```

Comment:
I'm just doing a quick drive-by review, but I wonder if it's a good idea to give a specific step number here (steps can move around or be removed over time, and people forget to update references like this one.

Reply:
Ahh! Good catch. That's a copy/pasta from my other PR

Reply:
Fixed. Thanks!